.github/workflows/deployment.yaml

```diff
@@ -0,0 +1,52 @@
+name: deployment
+
+on:
+  pull_request:
+    branches: [main]
```

Comment:
Let's get rid of the pull request bits here and below, and only do this on pushes to `main`.  We do pull request previews with Vercel already, so we know if it's working.